### DISCUSSION TITLE
Split Cub200 into more the five experiences.

### DISCUSSION THREAD

**Discussion by evertonaleixo** [ContinualAI/avalanche#1581] [Q&A]
🐛 **Describe the bug**
I am trying to split cub200 benchmark into 10 experiences, however, it is giving the following error:

'''
File ~\playground\avalanche\avalanche\benchmarks\scenarios\new_classes\nc_scenario.py:277, in NCScenario.__init__(self, train_dataset, test_dataset, n_experiences, task_labels, shuffle, seed, fixed_class_order, per_experience_classes, class_ids_from_zero_from_first_exp, class_ids_from_zero_in_each_exp, reproducibility_data)
    270 remaining_exps = n_experiences - len(per_experience_classes)
    271 if (
    272     remaining_exps > 0
    273     and (self.n_classes - sum(per_experience_classes.values()))
    274     % remaining_exps
    275     > 0
    276 ):
--> 277     raise ValueError(
    278         "Invalid number of experiences: remaining "
    279         "classes cannot be divided by n_experiences"
    280     )
    282 # default_per_exp_classes is the default amount of classes
    283 # for the remaining experiences
    284 if remaining_exps > 0:

ValueError: Invalid number of experiences: remaining classes cannot be divided by n_experiences
'''

🐜 **To Reproduce**
A [minimal working example](https://en.wikipedia.org/wiki/Minimal_working_example) to reproduce the issue. The code should be executable without modifications.
'''
split_ds = SplitCUB200(
    n_experiences=10,
    seed=1234,  
)
'''

🐝 **Expected behavior**
Create 10 experiences,

🐞 **Screenshots**

🦋 **Additional context**
Add any other context about the problem here like your python setup.

**Comment by tungts1101** [comment]
Did you resolve this issue?
